Footer component prototype from issue #172

pgc007 opened the pull request.

pgc007 committed:
Added <SVG to logo file

pgc007 committed:
work on footer

pgc007 committed:
Centred copyright text

Commit:
Made the following changes:

- Added the missing ';' after html character codes

- Remember all content should be inside the final container div closer (on line 45)

Commit:
Merge branch 'development' into paul

pgc007 committed:
Footer draft complete

Need heights of footer elements to complete

georgepstaylor reviewed (changes requested):
The file header.html does not exist on the Development branch - I'll update your branch to reflect this

georgepstaylor commented:
Blue part is 101px:
<img width="1207" alt="image" src="https://user-images.githubusercontent.com/3026030/83362966-73fe8e00-a38d-11ea-8ca3-3f087472577b.png">
Grey part is 54px:
<img width="1214" alt="image" src="https://user-images.githubusercontent.com/3026030/83362969-8aa4e500-a38d-11ea-84d2-501376a125b9.png">

Commit:
See description:

- Units are needed for sizings, (px, rem, vh etc)
- Font family should be declared at a higher level, rather than each class. (we'll do this once we know the font being used)
- Media queries were redundant as they made the same stylings.

georgepstaylor commented:
Issue link: https://github.com/alan-turing-institute/AutisticaCitizenScience/issues/172#issuecomment-636022384
